### PR TITLE
Feature/recommendations

### PR DESCRIPTION
**Pull request by AnirupPat**
This Pull request has the following changes: 

Frequently Bought Item Service Integration
Frequently Bought By Me Service Integration
Test cases for the above  implementation

### CODE REVIEW

File: src/app/components/home/home.component.ts [hunk 1]
```diff
@@ -96,9 +107,24 @@ export class HomeComponent implements OnInit {
...
-     this.recommendedSet()
+    // recommended prod
+    this.recommendedProductList()
+     //frequently bought
+     this.getFrequentlyBought().then(data => this.frequentlyBoughtSet(data))
+     //frequently bought by me
+     this.logInSession().then(data => {
+       if(data) {
+        this.getFrequentlyBoughtByMe().then(data => {
+          if(data === []){
```
Comment (reddy-s) [inline src/app/components/home/home.component.ts]: you can make use of the following
```suggestion
          if(data.length > 5){
```

File: src/app/components/home/home.component.ts [hunk 2]
```diff
@@ -96,9 +107,24 @@ export class HomeComponent implements OnInit {
...
+           this.carousel.FrequentlyBoughtByMe.title = "Frequently Bought by Me"
```
Comment (reddy-s) [inline src/app/components/home/home.component.ts]: Would be good to add such literals into constants.js

File: src/app/components/home/home.component.ts (hunk 1 above)
Comment (AnirupPat) [reply src/app/components/home/home.component.ts]: @reddy-s length property is not being recognised. And lets not restrict it to products more than 5

File: src/app/components/home/home.component.ts (hunk 2 above)
Comment (AnirupPat) [reply src/app/components/home/home.component.ts]: @reddy-s sure

File: src/app/components/footer/footer.component.html [hunk 3]
```diff
@@ -30,7 +30,8 @@ <h5 class="font-weight-bold mt-3 mb-4">Get to Know Us</h5>
             </li>
         </ul>
       </div>
-      <hr class="clearfix w-100 d-md-none">
+      <!-- let this be commented for now..social media page links  -->
```
Comment (AnirupPat) [inline src/app/components/footer/footer.component.html]: @reddy-s let this be commented for now..we need to use this at a later stage

**Pull request merged**